gta.go

```diff
@@ -237,15 +237,31 @@ func (g *GTA) markedPackages() (map[string]map[string]bool, error) {
 		return nil, fmt.Errorf("diffing directory for dirty packages, %v", err)
 	}
 
-	// we build our set of initial dirty packages from the git diff. The map
-	// value is true when the package was deleted.
+	// We build our set of initial dirty packages from the git diff. The map
+	// value is true when the package was deleted. The map keys are package
+	// import paths.
 	changed := make(map[string]bool)
+	embeddedChanged := make(map[string]struct{})
 	onlyTestsAffected := make(map[string]struct{})
 	onlyTestPackagesChanged := make(map[string]struct{})
 	for abs, dir := range dirs {
 		// TODO(bc): handle changes to go.mod when vendoring is not being used.
 
-		// ignore deleted directories that did not contain files.
+		// Add packages that embed the files of dir.
+		for _, f := range dir.Files {
+			// An embedded file may:
```

Comment:
Could you perhaps rephrase these list items? I think I understand the semantics, I just know I likely wont in a year or two. 😅

Reply:
What do you suggest?

Reply:
Had [to suggest it separately](https://github.com/digitalocean/gta/pull/31/files#r949693088) ☹️